_api-reference/ingest-apis/ingest-pipelines.md

```diff
@@ -0,0 +1,390 @@
+---
+layout: default
+title: Ingest pipelines
+parent: Ingest APIs
+nav_order: 10
+redirect_from:
+  - /opensearch/rest-api/ingest-apis/create-update-ingest/
+  - /api-reference/ingest-apis/create-update-ingest/
+  - /api-reference/ingest-apis/get-ingest/
+  - /api-reference/ingest-apis/simulate-ingest/
+  - /api-reference/ingest-apis/delete-ingest/
+---
+
+# Ingest pipelines
+
+Ingest pipelines in OpenSearch are managed by using ingest API operations. When using ingest in production environments, your cluster should contain at least one node with the node roles permission set to `ingest`. For more information about setting up node roles within a cluster, see [Cluster formation]({{site.url}}{{site.baseurl}}/opensearch/cluster/).
+
+## Prerequisites
+
+The following are prerequisites for using OpenSearch ingest pipelines:
+
+- <SME insert information>
```

Comment:
I think this line from description above could be one of prerequisites. `When using ingest in production environments, your cluster should contain at least one node with the `ingest` role.`

In addition to that, following prerequisites is needed when security features are enabled.
```
If the OpenSearch security features are enabled, you must have the `cluster_manage_pipelines` permission to manage ingest pipelines.
```

Reply:
Revised in PR #4299